components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -388,6 +417,27 @@ private void logSuccessResponse(Action action, ActionInvocationSuccessResponse s
         }
     }
 
+    private void logIncompleteResponse(Action action, ActionInvocationIncompleteResponse incompleteResponse) {
+
+        DIAGNOSTIC_LOGGER.logSuccessResponse(action);
```

Comment:
Is this correct? If the same functionality can be reused better rename

Reply:
Can we reuse the method functionality with diagnostic logger. 
This confuses the reader